vms/rpcchainvm/vm_server.go

```diff
@@ -843,3 +851,76 @@ func (vm *VMServer) StateSummaryAccept(
 		Err:  errorToErrEnum[err],
 	}, errorToRPCError(err)
 }
+
+func convertNetworkUpgrades(pbUpgrades *vmpb.NetworkUpgrades) (upgrade.Config, error) {
```

Comment:
`pbUpgrades` may be `nil` here - Can we error rather than panic in that case?

Reply:
added 👍